### DISCUSSION TITLE
Vercel can't connect to Github

### DISCUSSION THREAD

**Discussion by luenmuvel** [vercel/vercel#5351] [Ideas]
I have a React project that I'm trying to implement in Vercel, but I'm having a lot of trouble doing it:

1. I have a repository created on Github (with public access)
2. I have the permissions (on Github) on the specific repository for the Vercel application.
3. The Vercel system tells me that I already have the account connected, but then I see that it appears **"not connected"**.

![imagen](https://user-images.githubusercontent.com/34595961/97645678-9b7c8400-1a1b-11eb-8681-0f17484820f1.png)

![imagen](https://user-images.githubusercontent.com/34595961/97646169-01b5d680-1a1d-11eb-8c92-ff395714d904.png)

![imagen](https://user-images.githubusercontent.com/34595961/97645714-ae8f5400-1a1b-11eb-8e17-35bdadaed81f.png)

4. Every time I want to import the repository it sends me to install Vercel.
![imagen](https://user-images.githubusercontent.com/34595961/97646017-82c09e00-1a1c-11eb-878c-2be9e467292f.png)

5. And it sends me to Github to configure Vercel permissions. But I already did that before (step 2).

![imagen](https://user-images.githubusercontent.com/34595961/97645762-cc5cb900-1a1b-11eb-8751-ecc82111ffd6.png)

I already have another project hosted on Vercel, but I did it with GatsbyCloud and it was automatic, I didn't have to do anything. In this case I am doing it manually but I have many problems. On the other hand, I am using a free account at Vercel, does it have something to do with it?

**Comment by deepansharya1111** [comment]
Were you able to solve this issue?

**Reply by ydax** [reply]
I'm having this same issue

**Comment by ydax** [comment]
This must have been a cookies issue. When I logged into Vercel through an Incognito Window (Chrome), I was able to see the connected account / repo.

**Comment by Rounak-stha** [comment]
The problem still exists. Has anyone got any workarounds?

**Reply by leerob** [reply]
Can you give more details on what you're seeing?

**Reply by Rounak-stha** [reply]
It gives me this error. The page github.com/apps/vercel isn't woking. Maybe it's a github issue or something?

![Github_Unicorn](https://user-images.githubusercontent.com/66557682/201570793-a7ec69f3-d794-4421-bb11-b06d465a2a98.png)

**Reply by leerob** [reply]
Are you in the `cs50`/`me50` GitHub orgs by chance? There's a known issue on GitHub's side due to the size of these. If you leave those, it should work.

**Reply by Rounak-stha** [reply]
This fix worked. Thank you @leerob

**Reply by lucleray** [reply]
Hi @Rounak-stha. We just got the word from GitHub that they fixed this issue on their side. So that means that you should not see the 500 error anymore if you're contributor in the `cs50` or `me50` GitHub orgs 🎉
